CMakeLists.txt

```diff
@@ -150,7 +150,7 @@ else ()
     endif ()
 endif ()
 install(TARGETS "openrct2" RUNTIME DESTINATION "${CMAKE_INSTALL_BINDIR}")
-install(TARGETS "openrct2-cli" OPTIONAL RUNTIME DESTINATION "${CMAKE_INSTALL_BINDIR}")
```

Comment:
Why is this commented out?

Reply:
Something I have to revert, leftover from testing